Add new rule logging_services_active

mpurg opened the pull request:
#### Description:

- Checks that one of rsyslog/systemd-journald is active.
- Add rule to Ubuntu 24.04 CIS controls
- Fix metadata in rule `firewall_single_service_active`
 
#### Rationale:

- Satisfies Ubuntu 24.04 CIS control 6.1.1.4

dodys reviewed (commented):
@mpurg could you take a look at the failed tests?

mpurg commented:
Rebased and added cosmetic fix to tests.

Automatus fails are due to systemd not existing in container, same issue as #12511 
Works on KVM:
```
INFO - xccdf_org.ssgproject.content_rule_firewall_single_service_active
INFO - Script single.pass.sh using profile (all) OK
INFO - Script none.fail.sh using profile (all) OK
INFO - Script multiple.fail.sh using profile (all) OK
INFO - xccdf_org.ssgproject.content_rule_logging_services_active
INFO - Script none.fail.sh using profile (all) OK
INFO - Script multiple.pass.sh using profile (all) OK
INFO - Script journald.pass.sh using profile (all) OK
```

ctests:
```
100% tests passed, 0 tests failed out of 33
```

dodys reviewed (approved):
lgtm, thanks!
